sys/can/conn/isotp.c

```diff
@@ -366,7 +357,7 @@ int conn_can_isotp_close(conn_can_isotp_t *conn)
         }
     }
 #else
-    while (try_get_msg(conn, &msg)) {
+    while (mbox_try_get(&conn->mbox, &msg)) {
```

Comment:
IMO, a better change would be to use `try_get_msg()` above, instead of `mbox_try_get()`

Reply:
try_get_msg() was removed, because (as far as I could tell), the only purpose of that function was to make a distinction between MODULE_CONN_CANN_ISOTP_MULTI or not (single?). However, this check was already was already done where that function gets called. Thus, it seemed redundant. That's why I moved the code of the function try_get_msg() outside, and moved it in the preprocessor checks. 
It's always possible I made a rookie mistake ofcourse :)

Reply:
Yes, that's also fine actually.